eralchemy/models.py

```diff
@@ -87,28 +99,27 @@ class Relation(Drawable):
 
     @staticmethod
     def make_from_match(match):
-        return Relation(
-            right_col=match.group('right_name'),
-            left_col=match.group('left_name'),
-            right_cardinality=match.group('right_cardinality'),
-            left_cardinality=match.group('left_cardinality'),
-        )
+        return Relation(**match.groupdict())
 
-    def __init__(self, right_col, left_col, right_cardinality=None, left_cardinality=None):
+    def __init__(self, right_table, left_table, right_cardinality=None, left_cardinality=None, right_column=None, left_column=None):
         if right_cardinality not in self.cardinalities.keys() \
                 or left_cardinality not in self.cardinalities.keys():
             raise ValueError('Cardinality should be in {}"'.format(self.cardinalities.keys()))
-        self.right_col = right_col
-        self.left_col = left_col
+        self.right_table = right_table
+        self.right_column = right_column
+        self.left_table = left_table
+        self.left_column = left_column
         self.right_cardinality = right_cardinality
         self.left_cardinality = left_cardinality
 
     def to_markdown(self):
-        return "{} {}--{} {}".format(
-            self.left_col,
+        return '{}{} {}--{} {}{}'.format(
+            self.left_table,
+            '' if self.left_column is None else '."%s"' % self.left_column,
```

Comment:
Column names are wrapped in double quotes so that `.er` files generated prior to these changes can still be parsed correctly. Without some special column delimiter, it would be hard to distinguish the cases for `foo.bar`, where `foo` could either be a schema or table name, and `bar` could be a table or column name.